programmatic-usage/index.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
```

Comment:
Migrate this file into "tests" directory and rename as "run_mocha.js" (or similar).